.mailmap

```diff
@@ -0,0 +1,7 @@
+Aleksa Sarai <asarai@suse.de> <cyphar@cyphar.com>
+Brandon Philips <brandon.philips@coreos.com> <brandon@ifup.org>
+Doug Davis <dug@us.ibm.com> <duglin@users.noreply.github.com>
+Sergiusz Urbaniak <sur@coreos.com> <sergiusz.urbaniak@gmail.com>
+Stephen Day <stephen.day@docker.com> <stevvooe@users.noreply.github.com>
```

Comment:
From your [initial comment][1], @stevvooe uses “Stephen J Day” for his Docker address.  That's why I [suggested][2]:

    Stephen J Day <stephen.day@docker.com> <stevvooe@users.noreply.github.com>

Without the “J”, you get:

```
$ git describe
v0.2.0-43-g380a18c
$ git shortlog -se | grep Stephen
    27  Stephen Day <stephen.day@docker.com>
    15  Stephen J Day <stephen.day@docker.com>
```

With the line I suggest you get:

```
$ git shortlog -se | grep Stephen
    42  Stephen J Day <stephen.day@docker.com>
```

[1]: https://github.com/opencontainers/image-tools/pull/186#issue-260471725
[2]: https://github.com/opencontainers/image-tools/pull/186#pullrequestreview-65098077